src/Features/Onboarding.php

```diff
@@ -50,7 +50,12 @@ public static function get_instance() {
 	 */
 	public function __construct() {
 
+		// Adds the ability to toggle the new onboarding experience on or off.
+		// @todo We should consider when we want to remove the abillity to toggle this feature.
```

Comment:
I think this is something we could keep in for the 10% launch and 50% launch, and remove prior to merging this feature to core / 100% of users.

Reply:
That sounds right to me as well.  Updated the comment to be more specific about when this should be removed.